### DISCUSSION TITLE
Custom Short cuts

### DISCUSSION THREAD

**Discussion by sammd** [Anze/KeyCluCask#16] [Ideas]
Hi,

Would really like the ability to enter your own shortcuts. I use better touch tool and a gaming mouse with multiple function buttons.

I like where this app is heading and think this feature would put you above the rest. 
If you can get custom shortcuts to change with the active app/active window that would be icing on the cake.

Thanks in advance.

**Comment by Anze** [comment]
Hey sammd,
I'm not really sure if I got your idea right. You want to have possibility to set (in this case *change*) shortcut to custom within app, not using additional app or builtin macOS one? Or you want to have something different. Can you please explain in more details your idea, how you image it.

**Comment by Anze** [comment]
Closing discussion as feature was implemented. Adding custom shortcuts in Settings -> My Shortcuts. Also possible to download shortcuts defined by others from [KeyClu Extensions](https://github.com/Anze/KeyCluExtensions).
